Dockerfile

```diff
@@ -1,65 +1,76 @@
-# project is for the moment server only (no client in the same repo), so it deserves to be in the root dir
-
-FROM golang:1.18 as builder
-
-ENV DEBIAN_FRONTEND="noninteractive"
-RUN apt-get update
-RUN apt-get install -y \
-    build-essential \
-    git curl \
-    libboost-dev \
-    libcurl4-openssl-dev \
-    libuv1-dev \
-    libboost-all-dev \
+FROM golang:1.18-alpine as openvds
+RUN apk --no-cache add \
+    git \
+    g++ \
+    gcc \
+    make \
+    cmake \
+    curl-dev \
+    boost-dev \
     libxml2-dev \
-    libfmt-dev \
-    libssl-dev \
-    uuid-dev \
-    libpng-dev cmake
-
+    libuv-dev \
+    util-linux-dev
 
 WORKDIR /
 RUN git clone https://community.opengroup.org/erha/open-vds.git
 WORKDIR /open-vds
 RUN git checkout enable-cache
 
 RUN cmake -S . \
-          -B build \
-          -DCMAKE_BUILD_TYPE=Release \
-          -DBUILD_JAVA=OFF \
-          -DBUILD_PYTHON=OFF \
-          -DBUILD_EXAMPLES=OFF \
-          -DBUILD_TESTS=OFF \
-          -DBUILD_DOCS=OFF \
-          -DDISABLE_AWS_IOMANAGER=ON \
-          -DDISABLE_AZURESDKFORCPP_IOMANAGER=OFF \
-          -DDISABLE_GCP_IOMANAGER=ON \
-          -DDISABLE_DMS_IOMANAGER=OFF \
-          -DDISABLE_STRICT_WARNINGS=OFF
+    -B build \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_JAVA=OFF \
+    -DBUILD_PYTHON=OFF \
+    -DBUILD_EXAMPLES=OFF \
+    -DBUILD_TESTS=OFF \
+    -DBUILD_DOCS=OFF \
+    -DDISABLE_AWS_IOMANAGER=ON \
+    -DDISABLE_AZURESDKFORCPP_IOMANAGER=OFF \
+    -DDISABLE_GCP_IOMANAGER=ON \
+    -DDISABLE_DMS_IOMANAGER=OFF \
+    -DDISABLE_STRICT_WARNINGS=OFF
 RUN cmake --build build   --config Release  --target install  -j 8 --verbose
 
 
+FROM openvds as vdsslice
```

Comment:
Call it builder and explicitly builder? Then you differentiate build errors from test/install errors